### PR TITLE
Created new package xchain-thornode, auto generated api - draft

### PR DESCRIPTION
**Pull request by Thorian1te**
Created new package `xchain-thornode`
Auto generated api using openapi
Added relevant exports

### CODE REVIEW

File: packages/xchain-thornode/src/config.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const THORNODE_API_TS_URL = 'https://thornode.thorswap.net/';
+export const THORNODE_API_9R_URL = 'https://thornode.ninerealms.com/';
```
Comment (veado) [inline packages/xchain-thornode/src/config.ts:2]: Let users define custom urls by using `.env`. If it is not set in `.env`, then use `https://thornode.ninerealms.com` by default. All others can be removed from `config.ts`,  `https://thornode.thorchain.info` might be deprecated in the near future.

**Review by veado — DISMISSED** [review]
LGTM - just one comment.

File: packages/xchain-thornode/src/config.ts (hunk 1 above)
Comment (Thorian1te) [reply packages/xchain-thornode/src/config.ts:2]: yep sure thing. ill fix this now

**Review by thorchain-admin — COMMENTED** [review]
LGTM

**Pull request merged**